### DISCUSSION TITLE
Custom icon set ?

### DISCUSSION THREAD

**Discussion by KaKi87** [jarek-foksa/xel#119] [Q&A]
Hello,

I would like to use [Fork Awesome](https://github.com/ForkAwesome/Fork-Awesome) as icon set.

This isn't usually a problem, since it only requires :
```css
@import 'https://unpkg.com/fork-awesome@1.2.0/css/fork-awesome.min.css';
```
And :
```html
<i class="fa fa-ICON_NAME"></i>
```
But, when using `x-menuitem`, for example, adding this element will not replace the margin reserved for an `x-icon` component.
How to deal with that ?

Thanks

**Comment by jarek-foksa** [comment]
Xel supports only symbol-based SVG icon sprites, which you can reference via the `<x-icon>` element.

While you might be able to get "icon fonts" to work with it, you will very likely run into problems with incorrect sizing, positioning and coloring.
